Content.Client/GameObjects/Components/HUD/Inventory/ClientInventoryComponent.cs

```diff
@@ -240,5 +243,42 @@ public bool TryFindItemSlots(IEntity item, [NotNullWhen(true)] out Slots? slots)
 
             return false;
         }
+
+        [Verb]
+        private sealed class SetOutfitVerb : Verb<ClientInventoryComponent>
```

Comment:
I'd personally prefer if this was on the server instead. Should be the same, but changing `ClientInventoryComponent` to `InventoryComponent` on the server!